frontend/src/modules/FlowNetwork/view.tsx

```diff
@@ -32,11 +33,19 @@ export function View({ viewContext }: ModuleViewProps<Interfaces>) {
     }, [datedNetworks]);
 
     // Convert metadata lists to front-end format
-    const convertedEdgeMetadataList: EdgeMetadata[] = React.useMemo(() => {
-        return edgeMetadataList.map((elm) => ({ key: elm.key, label: elm.label }));
+    const convertedEdgeMetadataList = React.useMemo(() => {
+        return edgeMetadataList.map<EdgeMetadata>((elm) => ({
+            key: elm.key,
+            label: elm.label,
+            unit: simulationUnitReformat(elm.unit ?? ""),
```

Comment:
Not a big issue, but I'd prefer: `unit: elm.unit ? simulationUnitReformat(elm.unit) : ""` for readability.